src/main/webapp/src/app/app.module.ts

```diff
@@ -87,14 +88,16 @@ export function initGapi(gapiSession: GapiSession) {
         ],
       } as SocialAuthServiceConfig,
     },
-    {
-      provide: APP_INITIALIZER,
-      useFactory: initGapi,
-      deps: [GapiSession],
-      multi: true,
-    },
+    // Commented out as initGapi conflicts with SocialAuthService
```

Comment:
What's the plan here? To eventually resolve the conflict? If that's the intention, one roundabout way could be to reference an isolated commit that remove this section and its corresponding test case, and file and issue linking to that commit indicating that the code can be restored upon fixing the issue. 

Note that this is just a suggestion given the time constraints but in general it's not good practice to have commented code in your codebase. :)

Reply:
The plan is to eventually resolve the conflict. And time constraints prevent something more complicated from being done.

This reminds me of a humorous extreme version of this I stumbled across in some blog once: https://emacshorrors.com/posts/forget-me-not.html